sql/share/errmsg-utf8.txt

```diff
@@ -7361,6 +7361,9 @@ ER_NO_EIS_FOR_FIELD
 ER_WARN_AGGFUNC_DEPENDENCE
         eng "Aggregate function '%-.192s)' of SELECT #%d belongs to SELECT #%d"
         ukr "Агрегатна функція '%-.192s)' з SELECTу #%d належить до SELECTу #%d"
+ER_LOCKED_ACCOUNT
+        eng "Access denied, this account is locked"
+        rum "Acces refuzat, acest cont este blocat"
 
```

Comment:
Not here, but at the end of the file, please. And name it as in MySQL, `ER_ACCOUNT_HAS_BEEN_LOCKED`